src/controllers/MultiCacheController.ts

```diff
@@ -65,6 +65,74 @@ export default class MultiCacheController {
         .reflect();
     }
 
+    this.execAll(actions, reply);
+  }
+
+  public getFlushURLsConfig(cacheControllers: CacheController[]): Hapi.IRouteAdditionalConfigurationOptions {
+    return {
+      handler: this.flushURLs.bind(this),
+      tags: ['api', 'cache'],
+      description: 'Flush multiple caches',
+      validate: {
+        payload: this.getPayloadRequestSchema(cacheControllers, 'flushURLsConfig'),
+        failAction: (request: Hapi.Request, reply: Hapi.IReply, source: string, error: Boom.BoomError) => {
+          reply(ResponseBuilder.cleanupError(error));
+        },
+      },
+      plugins: {
+        'hapi-swagger': {
+          responses: {
+            200: {
+              description: 'All caches have been flushed',
+              schema: this.getSuccessResponseSchema(cacheControllers),
+            },
+            400: {
+              description: 'Bad Request',
+              schema: this.getBadRequestResponseSchema(cacheControllers),
+            },
+            502: {
+              description: 'Bad Gateway',
+              schema: this.getBadGatewayResponseSchema(cacheControllers),
+            },
+          },
+        },
+      },
+    };
+  }
+
```

Comment:
if i well understand, you don't flush the same thing for each CDN. is this due to CDN limitations? 
why do you give the full urls array for keycdn, but only one element for fasterize?

Reply:
Fasterize only support a single URL to be flushed at once.
It is safe here as there is validation (`min(1)`) on the input to prevent multiple URL to be flushed on Fasterize. We will remove this limitation later.